src/main/java/core/basesyntax/impl/WriterImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.service.util.Writer;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WriterImpl implements Writer {
+    private static final String BASE_DELIMITER = "=";
+    private static final String FILE_DELIMITER = ",";
+    private static final String TITLE_OF_REPORT = "fruit,quantity";
+    private static final String FILE_PATH_TO = "src/main/resources/fileTo.csv";
+    private final StorageDao storageDao;
+
+    public WriterImpl(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public void writeListToFile() {
```

Comment:
і куди писати теж краще не хардкодити, а передавати аргументом в метод
```suggestion
    @Override
    public void writeToFile(List<String> content, String fileName) {
```